src/text/span.rs

```diff
@@ -332,6 +332,12 @@ impl<'a> Span<'a> {
     pub fn to_right_aligned_line(self) -> Line<'a> {
         self.into_right_aligned_line()
     }
+
+    /// Returns `true` if the span is empty.
+    #[must_use]
+    pub fn is_empty(&self) -> bool {
```

Comment:
Is this actually used? I think this was used on a previous iteration and could be removed again / introduced in its own PR.

Reply:
Revoved in 65842dd1

Reply:
removed in 65842dd15